lib/actions/device.coffee

```diff
@@ -128,25 +128,40 @@ exports.register =
 		Examples:
 
 			$ resin device register MyApp
+			$ resin device register MyApp --uuid 83cb718aee3abe63d2e0a0016ea49399cdc5b3f9b879dbb01acf73f28f5085
+			$ resin device register MyApp --uuid 83cb718aee3abe63d2e0a0016ea49399cdc5b3f9b879dbb01acf73f28f5085 --device-api-key existingDeviceKey
```

Comment:
Ideas on what I should do about this? The example kinda needs to be this long :|